tests/unit/amo/helpers.js

```diff
@@ -214,18 +219,20 @@ export function dispatchSignInActions({
   authToken = userAuthToken(),
   userId = 12345,
   username = 'user-1234',
-  displayName = null,
+  // eslint-disable-next-line camelcase
+  display_name = null,
```

Comment:
Were you trying to avoid confusion here? Using camel case would help keep things consistent. It seems ok to me just to set like `display_name: displayName`

Reply:
Yeah, because this function/helper specifically creates an object that mimics an API response, I didn't like setting the `display_name` property with our (assigned/computed) `displayName` prop. It's really confusing to set `displayName` which will actually set `display_name` which is checked to set `displayName`. I thought the change disambiguated things; before then I found it hard to follow.